docker-java/src/test/java/com/github/dockerjava/cmd/CreateContainerCmdIT.java

```diff
@@ -1125,4 +1126,18 @@ public void shouldNotEncodeAuth() {
 
         assertThat(jsonNode.get("authConfig"), nullValue());
     }
+
+    @Test
+    public void shouldHandleANetworkAliasWithoutACustomNetworkGracefully() {
+        HostConfig config = newHostConfig()
+            .withPortBindings(new PortBinding(Ports.Binding.bindPort(8080), new ExposedPort(8081)));
```

Comment:
nit: I don't think this is relevant for this test, and, since it uses a fixed port, I would rather remote this line.

Reply:
Yes that's a good idea